pep/app/constants/preferences.ts

```diff
@@ -2,30 +2,39 @@ import ENV from 'pep/config/environment';
 import { ThemeId } from 'pep/constants/themes';
 import { LanguageCode } from 'pep/constants/lang';
 import { SearchTermId } from 'pep/constants/search';
+import Document from 'pep/pods/document/model';
 
 export enum PreferenceKey {
+    FAVORITES = 'favorites',
     THEME = 'theme',
     LANG = 'lang',
     SEARCH_LIMIT_IS_SHOWN = 'searchLimitIsShown',
-    SEARCH_TERM_FIELDS = 'searchTermFields'
+    SEARCH_TERM_FIELDS = 'searchTermFields',
+    READ_LATER = 'readLater'
 }
 
 export interface UserPreferences {
+    favorites?: Document[];
     preferencesVersion: string;
     theme: ThemeId;
     lang: LanguageCode;
     searchLimitIsShown?: boolean;
     searchTermFields?: SearchTermId[];
+    readLater?: Document[];
```

Comment:
While I agree with some of your points, I don't like the fact that we would have to keep making API calls to get the same data - over and over and over again. I could possibly compromise on loading id, and the text we need to display the items instead of the whole object.

Reply:
I agree, that if you say had 50 favorites and added a new favorite, it wouldnt be great to have to do an api call that reloads ALL 51 of those documents. We could make it smart tho w/o too much effort, by when their favorites update/change, using ember-data's cache and peeking for documents that already exist in the cache (which in this case would be all of them), and then only doing an api request for those that are not cached (which would really only happen on initial card load).

If we reaallly didnt want to try that though, at the very least yeah we should only be storing a "partial" document with the bare minimum fields needed to render the card (ID, title, author, etc). But knowning that the caveat here is we'd have to make concessions/logic in the code if they ever wanted to use additional fields here after release.

Also, does the reqs doc mention an upper limit anywhere for things like the user's number of Read Later or Favorite items? If not, we should probably discuss that too, as at the very least it should probably have some sane hard cap, not only for storage concerns but rendering concerns too.